src/relax/op/op.cc

```diff
@@ -177,7 +227,9 @@ TVM_REGISTER_OP("relax.call_builtin_with_ctx")
     .set_num_inputs(4)
```

Comment:
just noticed this should be 2, it is not related to your change...

Reply:
Good observation